modules/src/ping.py

```diff
@@ -1,14 +1,15 @@
 import requests
 
 from templates.text import TextTemplate
+from urlparse       import urlparse
 
 def process(input, entities):
     output = {}
     try:
         url = entities['url'][0]['value']
-        strip_list = ['http://', 'https://', '/']
-        for item in strip_list:
-            url = url.strip(item)
+        if '//' not in url:
+          url = '%s%s'%('//', url)
+        url = urlparse(url).hostname
```

Comment:
`hostname` returns `None` if it is unable to parse the url. See https://docs.python.org/2/library/urlparse.html Throw an error in that case.